source/features/tag-changes-link.tsx

```diff
@@ -33,8 +35,7 @@ async function getNextPage(): Promise<DocumentFragment> {
 }
 
 function parseTags(element: HTMLElement): TagDetails {
-	// Safari doesn't correctly parse links if they're loaded via AJAX #3899
-	const {pathname: tagUrl} = new URL($(['a[href*="/tree/"]', 'a[href*="/tag/"]'], element)!.href);
+	const tagUrl = expectElement(['a[href*="/tree/"]', 'a[href*="/tag/"]'], element).href;
```

Comment:
It still needs to work just on `pathname`

```suggestion
	const {pathname: tagUrl} = expectElement(['a[href*="/tree/"]', 'a[href*="/tag/"]'], element);
```

Reply:
You can see the pathname is empty in the console.
<img width="1631" alt="image" src="https://github.com/user-attachments/assets/97a7c17b-18d4-42fd-8d4f-3f7cc63de956">

Reply:
You can see the last dom is from next page. Because of some reasons its pathname is `''`.
<img width="1323" alt="image" src="https://github.com/user-attachments/assets/eec8e6ab-f77d-4a78-9412-6f351a602005">

```ts
```

Reply:
How about this?
```ts
const tagUrl = expectElement(['a[href*="/tree/"]', 'a[href*="/tag/"]'], element).href;
const tag = tagUrl.split('/').pop()!;
```

Reply:
This seems to be the same issue as https://github.com/refined-github/refined-github/issues/3899, except that it's in Chrome too.

It seems that tag names are escaped in URLs: https://github.com/refined-github/sandbox/releases/tag/sandbox%2Freleases%2Fsandbox%2Freleases%2F1

So it's probably fine to just use `.href`, but this needs to be documented:

```
// DO NOT change this to `pathname` because blah blah
```